pkg/ui/seeders.go

```diff
@@ -52,7 +52,7 @@ func (tui *TUI) SeedAckIncidentsUI() {
 	}
 
 	for _, i := range incidents {
-		incident := []string{i.Id, i.Title, i.Urgency, i.Status, i.Service.Summary}
+		incident := []string{i.Id, i.Title, i.Urgency, i.Status, i.Service.Summary, i.Assignments[0].Assignee.Summary}
```

Comment:
Can we follow the standard in i.Id, i.Title, i.Urgency, i.Status, i.Service.Summary and make i.Assignments[0].Assignee.Summary similar?
I'd suggest to keep it `i.Assignee.Summary`,  you can assign the value Assignments[0] to a variable and we can just use that. Let me know if this is not clear to you.